README.md

```diff
@@ -0,0 +1,47 @@
+[![Release](https://img.shields.io/github/v/release/opengisch/qgis-pg-service-parser-plugin.svg)](https://github.com/opengisch/qgis-pg-service-parser-plugin/releases)
+
+# PG service parser plugin
+
+<img src="https://raw.githubusercontent.com/opengisch/qgis-pg-service-parser-plugin/main/pg_service_parser/images/logo.png" alt="Logo" width="200px" align="left"/>
+
+
+QGIS v3 plugin to view, edit or copy PG service (i.e., `pg_service.conf`) entries for PostgreSQL connections.
+
+This plugin is distributed under the [GNU GPL v3 license](https://github.com/opengisch/qgis-pg-service-parser-plugin/blob/main/LICENSE).
+
+
+
+### pg_service.conf location
+
+Before using the plugin, make sure you've set the `PGSERVICEFILE` environment variable pointing to the `pg_service.conf` file path.
```

Comment:
Is this needed for linux too or does it have a default location?

Reply:
this is delegated to the lib https://github.com/opengisch/pgserviceparser, it might be better to state this?

Reply:
According to https://www.postgresql.org/docs/current/libpq-pgservice.html, we have both a per-user and a system default locations.

Per-user: `~/.pg_service.conf` or `%APPDATA%\postgresql\.pg_service.conf`
System: `/etc/postgresql-common/pg_service.conf `  (run `pg_config --sysconfdir` to get the directory).

On the other hand, even if the variable is not set, the plugin (in fact, the underlying lib) will attempt to find the `pg_service.conf` file at `~/pg_service.conf`.

So, we probably need to rephrase this line in the README.

The underlying lib could also be adjusted, so as to try to locate the file in the system location if not found in the per-user one.

Reply:
If there's a canonical doc in the lib, a link from here would be ok too.
Main point is, if it's documented here, better it is correct

Reply:
Right, I was playing safe for a first version of the README: Setting the variable will always allow the plugin to work predictably. However, let's rephrase it so that it's clear that if a user has a `.pg_service.conf` already in `~/`, she/he can avoid setting the variable.

I think the lib docs could be a bit hard to read for a regular user, so if you guys don't mind I'd state that here in the README.